src/transformers/modeling_longformer.py

```diff
@@ -1316,7 +1345,7 @@ def forward(
         return_dict = return_dict if return_dict is not None else self.config.use_return_dict
 
         # set global attention on question tokens
-        if global_attention_mask is None:
+        if global_attention_mask is None and input_ids is not None:
```

Comment:
(Curious) The `global_attention_mask` doesn't need to be computed when passing in `inputs_embeds`?

Reply:
good catch! I was being lazy haha -> will handle the `inputs_embeds` case as well